CLAUDE.md

```
@@ -0,0 +1,109 @@
+## Styling
+
+Use PatternFly 6 components and CSS variables:
+```typescript
+style={{ paddingBlockStart: 'var(--pf-t--global--spacer--md)' }}
+```
+
+Co-locate CSS files with components: `Dashboard.tsx` + `Dashboard.css`
+
+## Commit Message Format
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)